client/my-sites/hosting/main.js

```diff
@@ -205,6 +206,7 @@ class Hosting extends Component {
 
 		return (
 			<Main wideLayout className="hosting">
+				{ ! isLoadingSftpData && <ScrollToAnchorOnMount /> }
```

Comment:
It feels a little bit odd to have the scroll happen after the data loads. Should we have the scroll happen right away?

Reply:
We need to calculate the size of the first Card. That's why we perform the scroll after loading the data.
To avoid this, we could move the SFTP card to the latest element or set it to a fixed height independently of its content. I think the small delay it's better than the other options.
@danielbachhuber, Do you prefer another solution over the small delay?

Reply:
@sejas Could we attach a resize observer to the `SftpCard`, and re-scroll after the height changes?